src/v3/src/util/languageUtils.ts

```diff
@@ -21,8 +21,14 @@ export const loadLanguage = async (widgetProps: WidgetProps): Promise<void> => {
   const languageCode = getLanguageCode(widgetProps);
   const supportedLanguages = getSupportedLanguages(widgetProps);
 
+  // Remove trailing slashes from baseUrl
+  let assetsBaseUrl = baseUrl ?? '';
+  if (assetsBaseUrl[assetsBaseUrl.length - 1] === '/') {
+    assetsBaseUrl = assetsBaseUrl.substring(0, assetsBaseUrl.length - 1);
+  }
+
   return Bundles.loadLanguage(languageCode, i18n, {
-    baseUrl: baseUrl ?? '/',
```

Comment:
If `assets.baseUrl` equals `/`, widget will try to load language files from URL `http://labels/json/login_xx.json`
Trailing slashes should be removed same as for Gen2: https://github.com/okta/okta-signin-widget/blob/1858480328499bcd10c65fa53801d9bbb792aa18/src/models/Settings.ts#L620